src/Chiron/Chiron.fs

```diff
@@ -674,7 +659,7 @@ module Mapping =
 
         static member inline FromJson (_: DateTime) =
                 fun x ->
-                    match DateTime.TryParseExact (x, [| "r" |], null, DateTimeStyles.None) with
+                    match DateTime.TryParseExact (x, [| "r" |], null, DateTimeStyles.AssumeUniversal) with
```

Comment:
Turns out that even though the "r" format should always be UTC, parsing it assumes it's a local time. Thanks, ``System.DateTime``!

Although this is the kind of thing FsCheck is awesome for...